app/src/main/java/com/example/smarteden/greenhousefields/RecyclerViewAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.smarteden.greenhousefields
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.smarteden.R
+class CardViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+    val titleTextView= itemView.findViewById<TextView>(R.id.textview_title)
+    val subtitleTextView = itemView.findViewById<TextView>(R.id.textview_subtitle)
+    val descriptionTextView = itemView.findViewById<TextView>(R.id.textview_description)
+
+}
+class RecyclerViewAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```

Comment:
Da ViewHolder is normal innerhalb der adapter klasse

Reply:
moansd du mei klasse cardviewholder ? soll de in en recyclerviewadapter ?

Reply:
Ja

Reply:
De Namensgebung gfollt ma a no ned. RecyclerViewAdapter für wos? Do wad irgendwie sche, wenn irgendwos dabei steht von Felder oda sowos

Reply:
ok danke für de detect info des habi mi grod davor gfragt wie i de ganzen sachan oschaun ko